common/tools/dev-tool/src/commands/run/vendored.ts

```diff
@@ -64,5 +64,13 @@ export default async (...args: string[]): Promise<boolean> => {
     ),
   );
 
-  return executor(...args);
+  const status = await executor(...args);
+  if (!status) {
+    // opportunity to log recommendations for the commands
+    if (args[0] === "prettier" && args[1] === "--list-different") {
+      log.warn(`Invoke "rushx format" in package directory to fix formatting.`);
```

Comment:
Could we leave out Rush since we are transitioning and just say to invoke the format command?

Reply:
Sure, `npm run` should work too.